nixos/tests/cfssl.nix

```diff
@@ -22,7 +22,7 @@ import ./make-test-python.nix (
           serviceConfig = {
             User = "cfssl";
             Type = "oneshot";
-            WorkingDirectory = config.services.cfssl.dataDir;
+            inherit (config.systemd.services.cfssl.serviceConfig) WorkingDirectory StateDirectory StateDirectoryMode;
```

Comment:
This isn't actually necessary for the test to pass but I figured I'd fix it as well.